pkg/health/types.go

```diff
@@ -0,0 +1,11 @@
+package health
+
+// Units represents the units of a node returned by the '/units' endpoint.
+type Units struct {
```

Comment:
To avoid duplicating the response definition, we should directly use https://github.com/dcos/dcos-diagnostics/blob/master/api/structures.go#L16